lighthouse-plugin-publisher-ads/messages/en-US.json

```diff
@@ -76,10 +76,15 @@
       "description": "Moments of network idleness in the critical path to ad loading present opportunities to improve speed. Consider eliminating long tasks, timeouts, waiting on load events, or synchronous resources to avoid idleness. Chrome DevTools can be used to discover what is causing network idleness. [Learn more](https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference).",
       "displayValue": "%s s spent idle in critical path"
     },
+    "loads-gpt-from-sgdn": {
+      "title": "GPT tag is loaded from correct host",
+      "failureTitle": "Load gpt.js from correct host",
+      "description": "Load GPT from 'securepubads.g.doubleclick.net' to reduce GPT load latency. By ensuring that GPT is loaded from the same host as ad requests, browsers can avoid an additional DNS lookup and HTTP connection. Example:`<script async src=\"https://securepubads.g.doubleclick.net/tag/js/gpt.js\">`. [Learn more](https://pub-ads-audits.appspot.com/#loads-gpt-from-sgdn)."
```

Comment:
s/By ensuring that GPT is loaded from/By loading GPT from .../

It's good to avoid the passive voice